x-pack/plugins/ml/public/components/job_selector/job_selector_table/job_selector_table.test.js

```diff
@@ -84,7 +84,19 @@ const props = {
         fromPx: 1,
         label: 'Apr 17th 2019, 20:04 to May 18th 2019, 19:45',
         widthPx: 93.1
-      }
+      },
+    },
+    {
+      groups: ['test'],
+      id: 'timeseries-only-test-job',
+      isRunning: false,
+      isSingleMetricViewerJob: false,
+      job_id: 'timeseries-only-test-job',
```

Comment:
This `job_id` is a bit misleading for the test. Maybe something like `non-time-series-job` would be clearer, for the test which checks the cannot be selected if it is not a time series job.

Reply:
Good call - updating 👍